packages/i18n/src/ja.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'ja',
+  schema:           (issue) => `無効な型: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+  specific: {
+    bic:            (issue) => `無効なBIC: ${issue.received} を受け取った`,
+    bytes:          (issue) => `無効なバイト数: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+    creditCard:     (issue) => `無効なクレジットカード: ${issue.received} を受け取った`,
+    cuid2:          (issue) => `無効なCuid2: ${issue.received} を受け取った`,
+    custom:         (issue) => `無効な入力: ${issue.received} を受け取った`,
+    customAsync:    (issue) => `無効な入力: ${issue.received} を受け取った`,
+    decimal:        (issue) => `無効な10進数: ${issue.received} を受け取った`,
+    email:          (issue) => `無効なメールアドレス: ${issue.received} を受け取った`,
+    emoji:          (issue) => `無効な絵文字: ${issue.received} を受け取った`,
+    endsWith:       (issue) => `無効な末尾: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
```

Comment:
```suggestion
    endsWith:       (issue) => `無効な末尾: ${issue.expected} で終わることを期待したが、 ${issue.received} を受け取りました`,
```

I added the word `終わること` to make it clear that the end of a character is an error. However, when I look at other languages, it is not written in such detail, so I am confused about whether to express it in that way only in Japanese.

Reply:
> However, when I look at other languages, it is not written in such detail, so I am confused about whether to express it in that way only in Japanese.

Me too. I decided not to add additional messages such like the other languages.